tests/integration/test_docker.py

```diff
@@ -308,12 +308,12 @@ def test_set_name():
         assert cont.name
         cont.delete(force=True)
 
-        cmd = DockerRunBuilder(additional_opts=['--name', test_name])
-        cont = image.run_via_binary(cmd)
+        add_opt = ['--name', test_name]
+        cont = image.run_via_binary(additional_opts=add_opt)
         assert cont.name == test_name
         cont.delete(force=True)
 
-        cmd = DockerRunBuilder(additional_opts=['--name', test_name])
-        cont = image.run_via_binary_in_foreground(cmd)
+        additional_opts = ['--name', test_name]
+        cont = image.run_via_binary_in_foreground(additional_opts)
```

Comment:
Fails on `run_command_instance needs to be an instance of DockerRunBuilder`. (`additional_opts=additional_opts`)